README.md

```diff
@@ -89,32 +94,31 @@ curl 127.0.0.1:9900/search \
     -d '{"from": 0, "size": 0, "query": {"query_string": {"query": "+isTemplate:n"}}}' | jq -r '.hits.total.value'
 ```
 
-
-
 ### Development/debug
 
 Developments are made on https://github.com/geonetwork/geonetwork-microservices
 
-To run one service directly without docker, use the `local` profile.
+To run one service directly without docker, use the `standalone` profile.
```

Comment:
As far as I remember the `local` profile is not the same as `standalone` (maybe to check with Gabriel).
The point of the `local` profile (eg. https://github.com/geonetwork/geonetwork-microservices/blob/main/config/indexing-service.yml#L45) was to have the services infrastructure started with docker and to be able to work on a particular one locally (with debug mode, ...) connecting to the others. `standalone` which only apply to some services (mainly OGC API records) allows to start one service autonomously as a Spring boot app without the need of all Spring Cloud services. 
So starting the discovery service with `standalone` profile does not make much sense probably (BTW this profile not being declared in that module, it will probably start fine) but `local` is not "deprecated" I think.

Reply:
Hmm ok I did not have the history, thanks for the catch up.
I searched for `profiles: local` but didn't get any results, my bad cause it seems it's used !

Anyway, running `ogc-records` with local did not work so I think it needs a little bit of clarification.
I tried to update the doc accordingly, please check it again and give me your feedback.
Thanks